utils/handleToasts.ts

```diff
@@ -2,19 +2,25 @@ import { toast } from 'react-toastify';
 
 export function handleErrorToast(errorText: string) {
   try {
-    const toastId = toast(errorText, {
```

Comment:
Laittaisin vielä näin funktion alkuun ja käyttäisin `toastId` muuttujaa sen jälkeen. Näin vältetään se jos `errorText` olisi joskus jotain mikä löytyy id:nä jostain muualta. Aika pieni todennäköisyys mutta varmuuden vuoksi voisi olla parempi.
```ts
const toastId = 'toast-' + errorText;
```